docs/en/latest/FAQ.md

```diff
@@ -71,6 +71,10 @@ There are two possibilities when encountering slow luarocks:
 For the first problem, you can use https_proxy or use the `--server` option to specify a luarocks server that you can access or access faster.
 Run the `luarocks config rocks_servers` command(this command is supported after luarocks 3.0) to see which server are available.
 
+```shell
+luarocks install apisix --server https://luarocks.cn
```

Comment:
This command is wrong. We should wrap the server option in `make deps`: https://github.com/apache/apisix/blob/f657b9023b07091687a3163cb4de8594b0a0f593/Makefile#L61

Reply:
Strictly speaking, it is not wrong. But such a way to install APISIX's dependencies is not recommended. So it would be better to avoid using it in the doc.

Reply:
Hi @spacewander 

Ok, so what should I do?

Reply:
@stu01509 
See my first comment. "We should wrap the server option in `make deps` ...". We need to introduce something like `LUAROCKS_SERVER=https://luarocks.cn make deps`.

Reply:
@stu01509 
Hi, will you continue on this PR?

Reply:
Hi @spacewander 

Yeah, Please give me some time to understand

Reply:
Hi @spacewander 

Should I change to this?

```
### deps:             Installation dependencies
.PHONY: deps
deps: default
luarocks install apisix --server https://luarocks.cn
ifeq ($(LUAROCKS_VER),luarocks 3.)
	mkdir -p ~/.luarocks
ifeq ($(shell whoami),root)
	luarocks config variables.OPENSSL_LIBDIR $(addprefix $(OPENSSL_PREFIX), /lib)
	luarocks config variables.OPENSSL_INCDIR $(addprefix $(OPENSSL_PREFIX), /include)
else
	luarocks config --local variables.OPENSSL_LIBDIR $(addprefix $(OPENSSL_PREFIX), /lib)
	luarocks config --local variables.OPENSSL_INCDIR $(addprefix $(OPENSSL_PREFIX), /include)
endif
	luarocks install rockspec/apisix-master-0.rockspec --tree=deps --only-deps --local
else
	@echo "WARN: You're not using LuaRocks 3.x, please add the following items to your LuaRocks config file:"
	@echo "variables = {"
	@echo "    OPENSSL_LIBDIR=$(addprefix $(OPENSSL_PREFIX), /lib)"
	@echo "    OPENSSL_INCDIR=$(addprefix $(OPENSSL_PREFIX), /include)"
	@echo "}"
	luarocks install rockspec/apisix-master-0.rockspec --tree=deps --only-deps --local
endif
```

Reply:
Yes.

Reply:
Hi @spacewander 

I update the `Makefile` and revert `FAQ.md` files.